### PR TITLE
[batch][ui] let the web browser choose the table size and make / focus search

### PR DESCRIPTION
**Pull request by danking**
The CSS makes some strong statements about how the web browser should display it. I
removed all these statements which lets the web browser choose if it should wrap some
in-cell text, truncate the table, or expand the table.

I fixed some bad formatting and removed tabs (😱) from some pages.

Check my dev deploy: https://internal.hail.is/dking/batch

### CODE REVIEW

**Comment by danking** [comment]
Dev deploy still coming up...

File: web_common/web_common/templates/layout.html [hunk 1]
```diff
@@ -8,6 +8,16 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="{{ base_path }}/common_static/css/main.css" />
     {% block head %}{% endblock %}
+    <script src="//cdnjs.cloudflare.com/ajax/libs/mousetrap/1.4.6/mousetrap.min.js"></script>
+    <script type="text/javascript">
+      function search() {
```
Comment (jigold) [inline web_common/web_common/templates/layout.html:13]: I like my solution better than this one. PR forthcoming once I have a chance to dev deploy.

**Comment by konradjk** [comment]
I might bow out of review if only bc I can't see internal.hail.is

**Comment by danking** [comment]
Hmm. I thought we made you a full fledged developer. If you log in via https://auth.hail.is/login first does it work?

**Comment by danking** [comment]
dev deploy should be up for whoever reviews: https://internal.hail.is/dking/batch

**Comment by danking** [comment]
Looks shitty on mobile, but :shrug:.

**Comment by jigold** [comment]
I'm unauthorized to view this as well.

**Comment by danking** [comment]
blah right, you have to log into *my* auth.

**Comment by danking** [comment]
Well, I promise you it looks fine. We should figure out letting other devs log into dev deployed auth.

**Pull request merged**